src/main/java/org/folio/rest/service/AccountEventPublisher.java

```diff
@@ -37,7 +38,8 @@ public void publishAccountBalanceChangeEvent(Account account) {
   public void publishDeletedAccountBalanceChangeEvent(String accountId) {
     final Account account = new Account()
       .withId(accountId)
-      .withRemaining(new MonetaryValue(BigDecimal.ZERO));
+      .withRemaining(new MonetaryValue(BigDecimal.ZERO))
+      .withDateClosed(new Date());
```

Comment:
Could you please explain, why do we even need to add the value here, if it is not used later in the event payload?

Reply:
Actually at this point it is my mistake, since when deleting the Account, it does not require the closing date, because you are deleting, sorry

Submit a new commit to fix this https://github.com/folio-org/mod-feesfines/pull/213/commits/3e0d66752f02cfad3087c2b5653d7d8145fe68ef